### PR TITLE
Burgers executable

### PR DESCRIPTION
**Pull request by wthrowe**
### Types of changes:

- [ ] Bugfix
- [ ] New feature

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`. For
  instructions on how to perform the CI checks locally refer to the [Dev guide
  on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run `make doc`
  to generate the documentation locally into `BUILD_DIR/docs/html`. Then open
  `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

**Review by kidder — COMMENTED** [review]
@nilsdeppe please look at
@wthrowe does it give the right answer?

**Comment by kidder** [comment]
@nilsdeppe what are your thoughts about the proliferation of executables and example input files covering combinations of solutions, numerical fluxes, limiters and other compile time options

**Comment by wthrowe** [comment]
I tested and the parameters in the example input file give errors smaller than 2e-6, reaching a maximum around t=0.5 and then decaying.  (For some reason that I don't understand, a region of small error spreads outward from the center of the grid (where the solution is zero) and replaces the dominant region of error completely around t=9.)

File: src/Evolution/Executables/Burgers/EvolveBurgers.hpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+#include <vector>
+
+#include "Domain/DomainCreators/RegisterDerivedWithCharm.cpp"
+#include "Domain/Tags.hpp"
+#include "ErrorHandling/FloatingPointExceptions.hpp"
+#include "Evolution/Actions/ComputeVolumeDuDt.hpp"  // IWYU pragma: keep
+#include "Evolution/Actions/ComputeVolumeFluxes.hpp"  // IWYU pragma: keep
+#include "Evolution/DiscontinuousGalerkin/DgElementArray.hpp"  // IWYU pragma: keep
+// for LocalLaxFriedrichsFlux
```
Comment (nilsdeppe) [inline src/Evolution/Executables/Burgers/EvolveBurgers.hpp]: What is the comment to? Normally they're on the same as the include, if not could you specify e.g. `// Equations.hpp for Local`?

**Review by nilsdeppe — COMMENTED** [review]
In terms of testing different combinations:
- For the input file we could do something like what SpEC does where replacements are made. Maybe there's a configure file that tells a program/script what options to fill with what values. I'm not sure exactly what to do about expected output, though.
- For the hpp configurations I guess the same approach could work, but that would be more complicated since then we need to add compilation flags, etc.

Finally, I don't think we should hold up this PR to figure out how to deal with the tests :)

File: src/Evolution/Executables/Burgers/EvolveBurgers.hpp (hunk 1 above)
Comment (wthrowe) [reply src/Evolution/Executables/Burgers/EvolveBurgers.hpp]: Fixed.  I'd been having trouble with IWYU complaining about having other stuff on the line, but I've found a form it is happy with now.

**Review by nilsdeppe — APPROVED** [review]
Ah that's nice :) Good find!

**Review by kidder — APPROVED** [review]

**Pull request merged**